modules/watson-machine-learning/main.tf

```diff
@@ -10,11 +10,26 @@ module "cos" {
   cos_plan          = "standard"
 }
 
+module "storage_delegation" {
+  providers = {
+    ibm                           = ibm
+    ibm.deployer                  = ibm
+    restapi.restapi_watsonx_admin = restapi.restapi_watsonx_admin
+  }
+  source               = "git::https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da.git//storage_delegation?ref=v1.4.0"
+  count                = var.watsonx_project_delegated ? 0 : 1
```

Comment:
Don't we want to call that module when the storage is delegated (i.e. when we use KMS)? Then the  `0 : 1` should be reversed

Reply:
I just noticed this myself when the infrastructure pipeline failed, I did indeed have it backwards.